src/Tribe/Main.php

```diff
@@ -2642,6 +2642,7 @@ public function dateToTimeStamp( $date, $hour, $minute, $meridian ) {
 		 * @todo remove - unused
 		 */
 		protected function dateHelper( $date ) {
+			_deprecated_function( __FILE__, '3.11', 'date' );
```

Comment:
The `dateToTimeStamp` function is `public`, though - so we'll keep it around for a bit.